tools/run_tests/run_tests.py

```diff
@@ -178,22 +178,18 @@ def _is_use_docker_child():
     'venv_relative_python',
     'toolchain',
     'runner',
-    'test_name',
-    'iomgr_platform',
 ])
 
 
 def _python_config_generator(name, major, minor, bits, config_vars):
-    name += '_' + config_vars.iomgr_platform
-    return PythonConfig(
-        name, config_vars.shell + config_vars.builder +
-        config_vars.builder_prefix_arguments +
-        [_python_pattern_function(major=major, minor=minor, bits=bits)] +
-        [name] + config_vars.venv_relative_python + config_vars.toolchain,
-        config_vars.shell + config_vars.runner + [
-            os.path.join(name, config_vars.venv_relative_python[0]),
-            config_vars.test_name
```

Comment:
Looks like `config_vars.test_name` is removed. Reasoning?

Reply:
`config_vars.test_name` was not a good name, it actually represents "test command" we wrote in `commands.py`. Python is special even in run_tests, we have several intermediate data class, like "PythonConfig", which used as a holder for the build command, and part of the run-single-test-case command. This PR moves the concatenation of the last argument to `def test_specs` (line 649).